src/plonemeeting/portal/core/content/item.py

```diff
@@ -95,14 +95,15 @@ def get_pretty_representatives(object):
     res = []
     institution = api.portal.get_navigation_root(object)
     mapping = institution.representatives_mappings
+    representatives_mapping = {}
     for infos in mapping:
-        representative_key = infos["representative_key"]
-        if representative_key in representative_keys:
-            representative_keys.remove(representative_key)
-            res.append(infos["representative_value"])
+        representatives_mapping[infos["representative_key"]] = infos["representative_value"]
```

Comment:
@odelaere 
tu peux faire un "dict comprehension" :
`representatives_mapping = {infos["representative_key"]: infos["representative_value"] for infos in mapping}`